### PR TITLE
Release / 4.3

### PR DESCRIPTION
**Pull request by zuzunker**
Milestone: https://github.com/Emurgo/yoroi-mobile/milestone/2?closed=1

## Initial Change Set

- [x] https://github.com/Emurgo/yoroi-mobile/pull/1414 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1421 [@v-almonacid]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1451 [@vsubhuman]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1453 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1454 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1456 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1459 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1460 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1461 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1462 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1471 [@vsubhuman]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1472 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1475 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1477 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1479 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1480 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1481 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1482 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1483 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1484 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1485 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1486 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1487 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1488 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1489 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1490 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1491 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1492 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1493 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1494 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1495 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1496 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1497 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1498 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1499 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1500 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1501 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1502 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1503 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1504 [@stackchain]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1505 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1506 [@wolverineks]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1509 [@v-almonacid]
- [x] https://github.com/Emurgo/yoroi-mobile/pull/1544 [@wolverineks]

## Release Notes

- Ledger delegation fix
- Translations updates
- Few small UI improvements
- Internal code improvements

### CODE REVIEW

**Review by zuzunker — COMMENTED** [review]
/check

**Review by zuzunker — COMMENTED** [review]
/check

File: package.json [hunk 1]
```diff
@@ -90,7 +90,7 @@
     "react": "16.13.1",
     "react-intl": "^2.8.0",
     "react-markdown": "4.3.1",
-    "react-native": "0.63.4",
+    "react-native": "0.64.2",
```
Comment (v-almonacid) [inline package.json:93]: @vsubhuman note that in general, it's not recommended to just bump the RN version without considering other modifications in the project that are included in the new target version.

The recommended way is to manually include all these changes (but maybe not all are critical or even necessary at all): https://react-native-community.github.io/upgrade-helper/?from=0.63.4&to=0.64.2

Comment (zuzunker) [reply package.json:93]: Thank you, Vicente!

**Pull request closed without merging**